app/src/main/java/edu/agh/susgame/front/gui/components/common/theme/Animations.kt

```diff
@@ -0,0 +1,52 @@
+package edu.agh.susgame.front.gui.components.common.theme
+
+import edu.agh.susgame.R
+
+
+object Animations {
+    data class AnimationFrames(val frames: List<Int>, val frameRate: Long)
+    data class AnimationFramesString(val frames: List<String>, val frameRate: Long)
+
+    val LOADING_ANIMATION = AnimationFrames(
+        frames = listOf(
+            R.drawable.i0,
+            R.drawable.i1,
+            R.drawable.i2,
+            R.drawable.i3,
+            R.drawable.i4,
+            R.drawable.i5,
+            R.drawable.i6,
+            R.drawable.i7,
+            R.drawable.i8,
+            R.drawable.i9,
+            R.drawable.i10,
+            R.drawable.i11,
+            R.drawable.i12,
+            R.drawable.i13,
+            R.drawable.i14,
+            R.drawable.i15,
+            R.drawable.i16,
+            R.drawable.i17,
+            R.drawable.i18,
+            R.drawable.i19,
+            R.drawable.i20,
+            R.drawable.i21,
+        ), frameRate = 24
```

Comment:
Hmm, z tego co widzę można tego uniknąć używając Javowej refleksji:
```kotlin
val LOADING_ANIMATION = AnimationFrames(
    frames = (0..21).map { id ->
        val resourceName = "i$id"
        val resourceId = R.drawable::class.java.getField(resourceName).getInt(null)
        resourceId
    },
    frameRate = 24,
)
```

Nie jest to też rozwiązanie idealnie, ale IMO warto